torchx/c_src/torchx.cpp

```diff
@@ -129,15 +137,22 @@ create_tensor_resource(ErlNifEnv *env, torch::Tensor tensor)
   ret = enif_make_resource(env, tensorPtr);
   enif_release_resource(tensorPtr);
 
+  printf("Tensor %p is created\n", tensorPtr);
```

Comment:
I think this should be using std::cout if this is going to be kept in the code

```suggestion
  std::cout << "Tensor "<< tensorPtr << " is created" << std::endl;
```

Reply:
this is just for quick debug. I will remove it after I sort out the problem

Reply:
Don't mind me then haha!